django/contrib/sessions/backends/signed_cookies.py

```diff
@@ -17,7 +17,9 @@ def load(self):
                 # This doesn't handle non-default expiry dates, see #19201
                 max_age=settings.SESSION_COOKIE_AGE,
                 salt='django.contrib.sessions.backends.signed_cookies')
-        except (signing.BadSignature, ValueError):
+        except Exception:
+            # BadSignature, ValueError, or unpickling exceptions. If any of
+            # these happen, reset the session.
```

Comment:
Are you okay to defer it until someone reports an actual problem? Also, I question the usefulness of adding more logging until we change Django's default logging config so logged statements are actually displayed somewhere.